spec/serializers/federation_entity_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe FederationEntitySerializer do
+  class TestEntity < DiasporaFederation::Entity
+    property :test, :string
+  end
+
+  let(:object) { double }
+
+  def enable_test_entity_builder
+    allow(Diaspora::Federation::Mappings).to receive(:builder_for).with(object.class).and_return(:test_entity)
+    allow(Diaspora::Federation::Entities).to receive(:test_entity).with(object) {
+      TestEntity.new(test: "asdf")
+    }
+  end
+
+  before do
+    enable_test_entity_builder
```

Comment:
Well, I followed the idea of self-description here: instead of writing a comment explaining what the specific piece of code does I extract it to a method, which name explains what it does. But if you think I should change it, I'm okay with it.